### PR TITLE
fix dead link to server option

### PR DESCRIPTION
**Pull request by kmaehashi**
Fixed dead link in FAQ.

### CODE REVIEW

File: source/en/faq.rst [hunk 1]
```diff
@@ -96,12 +96,12 @@ RPC Errors
 
 - Client library occasionally throws RPC timeout errors; it seems that servers automatically disconnect clients. Why?
 
- Jubatus servers automatically close connections when the idle timeout (given by the command line parameter :option:`server -t`) expires.
+ Jubatus servers automatically close connections when the idle timeout (given by the command line parameter :option:`jubatus_server -t`) expires.
```
Comment (rimms) [inline source/en/faq.rst]: I think we should write about proxy (`jubatus_proxy -t`) .

**Comment by rimms** [comment]
@kmaehashi Reviewed.

**Comment by kmaehashi** [comment]
Raised another issue for that. #267

**Comment by rimms** [comment]
Thanks. I will merge this PR.

**Pull request merged**